README.md

```diff
@@ -1,786 +1,104 @@
-[Wikipedia:List of ISO 639-3 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-3_codes).
-Alternatively, a language's 2-character
-[ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1) code may be used. A list
-of ISO 639-1 codes is maintained at
-[Wikipedia:List of ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).
+**The _Citation File Format_ is a collaborative project** and we welcome suggestions and contributions. We hope one of the invitations below works for you, but if not, please let us know!
 
-Example for a work in both English and Daakaka:
+:running: **I'm busy, I only have 1 minute**
+- Tell a friend about the *Citation File Format*!
 
-```yaml
-references:
-  - type: book
-    ...
-    languages:
-      - en
-      - bpa
-```
+:hourglass_flowing_sand: **I've got 5 minutes - tell me what I should do**
+- Create a `CITATION.cff` file for your repository.
+- Suggest ideas for how you would like to use the *Citation File Format*, or for an improvement to the format or its tooling.
 
-## Identifier type strings
+:computer: **I've got a few hours to work on this**
+- Take a look at the issues and see if there are any you can contribute to.
 
-The key
+:tada: **I really want to help increase the community**
+- Organise a hack event or workshop to help others use or improve the *Citation File Format*.
 
-```yaml
-identifiers:
-  - type
-```
-
-can only take the following values:
-
-- `doi`: Signifies that the `value` string of the identifier typed thus is a valid DOI.
-- `url`: Signifies that the `value` string of the identifier typed thus is a valid URL.
-- `swh`: Signifies that the `value` string of the identifier typed thus is a valid [Software Heritage identifier](https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html).
-- `other`: Signifies that the `value` string of the identifier typed thus is a valid identifier not currently known to CFF.  
-If you want to add an identifier type to CFF, please [create a new issue on the CFF GitHub repository](https://github.com/citation-file-format/citation-file-format/issues/new), and suggest a name for the identifier, and ideally also describe its format as a valid regex.
-
-Examples for valid identifiers:
-
-```yaml
-identifiers:
-  - type: "other"
-    value: "other-schema://abcd.1234.efgh.5678"
-  - type: "swh"
-    value: "swh:1:rel:99f6850374dc6597af01bd0ee1d3fc0699301b9f"
-```
+Please read the more detailed [contributing guidelines](CONTRIBUTING.md) and [open a GitHub issue](https://github.com/citation-file-format/citation-file-format/issues) to suggest a new idea or let us know about bugs.
 
-# Schema
+## License :balance_scale:
 
-CFF `CITATION.cff` files can be validated against a schema which is available at
-<https://github.com/citation-file-format/citation-file-format/blob/master/schema.yaml>.
+Copyright © 2016ff. The Citation File Format Contributors
 
-# Contributions
+This work is licensed under multiple licenses:
```

Comment:
Not sure that's technically true, but different parts of the repo are licensed differently. Maybe we should write down somewhere why we chose to have the mix, because I've forgotten ...again

Also the Zenodo metadata for 1.1.0 has yet a different license (CC-BY-SA-4.0; #245)